src/moderation/moderation.service.ts

```diff
@@ -27,4 +27,56 @@ export class ModerationService {
                     !blockedAccountIds.has(post.repostedBy.id)),
         );
     }
+
+    async filterUsersForPost(
+        userIds: number[],
+        post: Post,
+        repostedBy?: number,
+    ) {
+        // Map user ids to their corresponding account ids
+        const userAccountMap = new Map<number, number>();
+
+        for (const row of await this.db('users')
+            .whereIn('id', userIds)
+            .select('id', 'account_id')) {
+            userAccountMap.set(row.id, row.account_id);
+        }
```

Comment:
```suggestion
        const rows = await this.db('users')
            .whereIn('id', userIds)
            .select('id', 'account_id'));

        for (const row of rows) {
           userAccountMap.set(row.id, row.account_id);
        }
```

Reply:
Maybe a lil easier to parse?

Reply:
Yeh, don't know why I decided to do it this way 🙃

Fixed in https://github.com/TryGhost/ActivityPub/pull/572/commits/755d99fd4a5739566c595c6742f172bcc0333ddc